improver_tests/acceptance/SHA256SUMS

```diff
@@ -553,33 +552,31 @@ efff8720d452591b6531a279bad6c0168db2336fa497c1f1087ecfcb9321af92  ./sleet_probab
+d4dcdeb1ed121db2c9e96db11283d20a80ab5924ee82b750cab8b02342e16249  ./spot-extract/inputs/all_methods_uk.nc
+5a5e97eeaf2fd457f0e2b655e1b5a0e4befc0176fa1d45d04e7c7439ea9ccedb  ./spot-extract/inputs/all_methods_uk_unique_ids.nc
+1f83848537fb5b7fa11708ed3b12966d8c6850fc97f2616411646e7a61ea4a7d  ./spot-extract/inputs/enukx_lapse_rate.nc
+a71031207447765f56f1c6a1d03342bfbd4363f4f170d191eb690c54d3fe8082  ./spot-extract/inputs/enukx_preciprate_realizations_thresholds.nc
+ddbbef39c8ee888476ed677cadc003ccffc15d01d73ac5e7874c16084781c716  ./spot-extract/inputs/enukx_temperature_percentiles.nc
+6d40d71b5d53c5ed8ad6dd3432c38b285b044d5cc44fa106203aa2fd679341d5  ./spot-extract/inputs/enukx_temperature_realizations.nc
+bd3133e846f2c40b85d618eb0c09979aeaedc39aad22aa1460dedb860dadae81  ./spot-extract/inputs/enukx_temperature_thresholds.nc
+b5de3f80c0bd4c79bf77c4065fd1b37bab69f99ed2f57af5d54c371faabdf151  ./spot-extract/inputs/enukx_temperature_thresholds_multi_time.nc
+7b752b31d3d0d1444e63cb64499e2aa1bf32d7b901952c0751530f5ac847b841  ./spot-extract/inputs/nearest_uk.nc
+e5c9928e499c3479e721febee57b90e822d22f4e61f928027d9558a1ab7e48bc  ./spot-extract/inputs/ukvx_lapse_rate.nc
+9f4d1b8546bccf6a41ed4effbefe2514557849fc664dbb5e53b8cb9855baf03c  ./spot-extract/inputs/ukvx_lapse_rate_2m.nc
+0c9ecc81ac78a94fea6cc2ac5547625e0093b4b4007a03feed711fcebf8e4fa0  ./spot-extract/inputs/ukvx_lapse_rate_no_height.nc
+deb86e91c4d2d63ce947e8fd830194704d714cbf12d4759975160fc77283aa90  ./spot-extract/inputs/ukvx_pmsl.nc
+e0e675885f8aaa40fb3d4cb4e025bf0010110ef2bf7589bba6e3eb7c7e6c7547  ./spot-extract/inputs/ukvx_temperature.nc
+990fa6216bc485e7b95005acef4eb66a7de8ec1dd3d8b49d99770375ea6f76c6  ./spot-extract/outputs/extract_multiple_percentiles_kgo.nc
+f5b76324e319dc1c27aaca76eb5f1299820cb5317e21f3fe23be8611e6db5eda  ./spot-extract/outputs/extract_percentile_kgo.nc
+c14e6b835baa4427ad129a5dc3bfbf47b28400527a90013e1240c2f158313bdc  ./spot-extract/outputs/extract_resampled_percentiles.nc
+6339135535fd59756df6c91665b4cfdc92d8dd1527e600363d6067feac682b0c  ./spot-extract/outputs/lapse_adjusted_multiple_percentile_kgo.nc
+ec190e3249be6588f54d44061038257e86a9f3f9fee2e1690ddcc1315f9dd379  ./spot-extract/outputs/lapse_rate_adjusted_uk_temperatures.nc
+8acde1c5b6e152e71fcfe981dae3e425d8b59c891997c998f9d6cb2cf67a6ac4  ./spot-extract/outputs/mindz_land_constraint_uk_temperatures.nc
+d24b59b25252796d6ee2f26e7e6764b84145957b0d32a4246e075db931b7b46c  ./spot-extract/outputs/mindz_uk_temperatures.nc
+332165b98af3eac6cd56c30457cc9c031b9788a1f660295a06b0108b40816b57  ./spot-extract/outputs/multi_time_kgo.nc
+4cf5c380cff65016b32bd852e31a9ed3ea4ff60fcf879e75f760c63fac27c58d  ./spot-extract/outputs/nearest_uk_temperatures.nc
+5d68bab5444b1ee2c1ad6179bf19ba58cfc1cea60bc9684b0100c5c0035d4b74  ./spot-extract/outputs/nearest_uk_temperatures_unique_ids.nc
+23007b89be636fb0e7445f239a30fc9debf530fc19e33599a09226ec31625e7e  ./spot-extract/outputs/with_realization_collapse.nc
```

Comment:
This output file contains only zeroes. So did the previous version. Basically, it isn't raining in the source data. Can you replace the source data so this task does something meaningful?